index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Randall Mowen | INFO 2439 | Mobile Home Page</title>
+    <meta charset="utf-8">
+    <link rel="stylesheet" type="text/css" href="_css/Mowen_a1.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css" />
+    <script src="http://code.jquery.com/jquery-1.11.1.min.js"></script>
+    <script src="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+</head>
+<body>
+<!-- Randall Mowen
+     index.html
+     INFO2439
+     Thoendel
+     9/5/19
+-->
+
+<div data-role="page" id="homePage">
+    <header>
+    <div data-role="header" data-position="fixed" data-id="header" class="top-header" data-theme="a"><h1>Randall Mowen</h1></div>
+    </header>
+
+    <div data-role="main" class="ui-content">
+        <h1>INFO 2439</h1>
+        <h2>Fall 2019</h2>
+            <div data-role="header"><h3>Homework Assignments</h3></div>
+            <ul data-role="listview" data-inset="true">
+                <li><a href="module 2/Assignment2.html" rel="external">Assignment 2</a></li>
```

Comment:
If your a2 css doesn't seem to be applying until a refresh - try adding data-ajax="false" as an attribute to your <a> tag here. That will disable ajax for the link, forcing a full refresh of the css.